core/examples/eval_js_value.rs

```diff
@@ -10,6 +10,11 @@ use deno_core::v8;
 use deno_core::JsRuntime;
 use deno_core::RuntimeOptions;
 
+// This is a hack to make the `#[op]` macro work with
+// deno_core examples.
+// You can remove this:
+use deno_core::*;
```

Comment:
Is this hack required or not? If not it should be removed, if it is required then it's another red flag for me. We shouldn't require `deno_core::*` import

Reply:
This is only for examples residing in the `deno_core` crate.